test/cli/commands_test.py

```diff
@@ -29,6 +30,31 @@
 re_ansi_escape = re.compile(r"\x1b[^m]*m")
 
 
+@pytest.fixture()
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
```

I think if you make this change, it'll automatically apply this fixture to every test in this `.py` file without you having to explicitly add it to each test. (This can be handy because it means future tests added to this file will automatically get the same behavior.)

https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request

Reply:
I tried that and it didn't work 🤔 . I'll have another go - but if not then I'll just merge as is.

Reply:
No worries. We can always do it later. I've successfully used `autouse` fixtures a few times, so there may be something subtle going wrong. They are definitely a bit magical, which is both good and bad. 😬